pages/DSNP/Announcements.md

```diff
@@ -40,6 +40,16 @@ Additional duplicate Announcements MUST be rejected or ignored.
 Announcements may not be deleted, but some may be marked as invalid by using a [Tombstone Announcement](Types/Tombstone.md), or updated by using an [Update Announcement](Types/Update.md).
 For example, if a user creates a Reaction Announcement, they may remove that reaction by creating a Tombstone Announcement.
 
+## Operations
+
+Each of the these MUST have verifiable approval of the acting
+principal via signature proof or other system.
```

Comment:
Here and elsewhere, we have tried to stick with one sentence per line (any length) to make diffs easier. (Here and on `Identity.md`). I don't think we have a lint to enforce that however as it is hard to know what is a sentence.

```suggestion
Each of the these MUST have verifiable approval of the acting principal via signature proof or other system.
```